assignment_6.py

```diff
@@ -0,0 +1,59 @@
+# Rearranging cars
+
+def rearrange_cars(cars,order):
+    '''
+        Input: - 2 list of integers
+                cars =  a list of integers that represents the cars
+                order = a list of integers that represents in which order should the cars be rearranged
+        Returns a list that contains all the moves made to get the rearrangement        
+                
+    '''
+    empty = find_zero(cars) 
+    moves = []
+    dic = build_dic(order)
+    for i in range(len(cars)):
+        if cars[i] != order[i] and cars[i]!= 0:
+            car_pos = dic[cars[i]]      
+            
+            if car_pos != empty:
+                moves.append('move from {} to {}'.format(empty, car_pos))
+                
+            cars[empty], cars[car_pos] = cars[car_pos], cars[empty]
+            empty = car_pos
+            moves.append('move from {} to {}'.format(empty, i)) 
+    
+            cars[i], cars[empty] = cars[empty], cars[i]
+            empty = i
+
+    return moves   
+            
+
+def build_dic(lst):
+    '''
+        Input: a list
+        It bulids a dictonary from the list 
+    '''
+    dic = {}
+    for i in range(len(lst)):
+        dic[lst[i]] = i
+    return dic
+
+def find_zero(lst):

```

Comment:
I think python list **index()** method does a similar thing, maybe you could use it